### PR TITLE
refactor/useRemotePMandMHFR

### PR DESCRIPTION
**Pull request by danielmwakanema**

### CODE REVIEW

File: src/modules/facilities/master-health-facility.client.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import Axios, { AxiosError } from 'axios';
+import { Wrapper } from "../config/wrapper";
+import { LOGGER } from '../logging/winston.logger';
+import { Facility } from "./facility.interface";
+import { MasterHealthFacilityClientGatewayException } from './master-health-facility-client.exception';
+
+export class MasterHealthFacilityClient {
+    async findFacilityByCode(code: string): Promise<Facility|null> {
+        const host = Wrapper.get<string>('AVW_MHFR_URL');
```
Comment (bwighane) [inline src/modules/facilities/master-health-facility.client.ts:9]: This env (AVW_MHFR_URL) variable is missing in the .default.env committable file, just also verify for any newly added ones.

File: src/modules/products/product-master.client.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import Axios, { AxiosError } from 'axios';
+import { Wrapper } from "../config/wrapper";
+import { LOGGER } from '../logging/winston.logger';
+import { ProductMasterClientGatewayException } from './product-master-client-gateway.exception';
+import { Product } from "./product.interface";
+import { getProductMasterSrc } from './helpers';
+
+export class ProductMasterClient {
+    async findProductByCode(code: string, client: string): Promise<Product|null> {
+        const host = Wrapper.get<string>('AVW_PRODUCT_MASTER_URL');
```
Comment (bwighane) [inline src/modules/products/product-master.client.ts:10]: Missing env key (AVW_PRODUCT_MASTER_URL) in default.env file.

**Review by bwighane — CHANGES_REQUESTED** [review]

File: src/modules/facilities/master-health-facility.client.ts (hunk 1 above)
Comment (danielmwakanema) [reply src/modules/facilities/master-health-facility.client.ts:9]: Noted and fixed.

File: src/modules/products/product-master.client.ts (hunk 2 above)
Comment (danielmwakanema) [reply src/modules/products/product-master.client.ts:10]: Noted and fixed.

**Review by bwighane — APPROVED** [review]

**Pull request merged**